spec/models/spree/payment_method_spec.rb

```diff
@@ -28,16 +28,20 @@ module Spree
     end
 
     it "computes the amount of fees" do
-      pickup = create(:payment_method)
       order = create(:order)
-      expect(pickup.compute_amount(order)).to eq 0
-      transaction = create(:payment_method, calculator: Calculator::FlatRate.new(preferred_amount: 10))
-      expect(transaction.compute_amount(order)).to eq 10
-      transaction = create(:payment_method, calculator: Calculator::FlatPercentItemTotal.new(preferred_flat_percent: 10))
-      expect(transaction.compute_amount(order)).to eq 0
+
+      free_payment_method = create(:payment_method) # flat rate calculator with preferred_amount of 0
```

Comment:
This is much better. Next time I would be tempted to convert the comment into code as well. Passing on the calculator may not be necessary, but if we rely on the calculator, we shouldn't trust the factory default. It would make the comment obsolete and still work if the default changed.

Reply:
yes agree.
in this case this is not a default factory, it's [a default value in the object](https://github.com/openfoodfoundation/openfoodnetwork/blob/c178478a4f0210133e98e0fbcc25a4d96943a300/app/models/spree/payment_method_decorator.rb#L47).
maybe I should have called it default_free_payment_method...